### DISCUSSION TITLE
SONOFF ZBMINI-L

### DISCUSSION THREAD

**Discussion by mprice43** [Koenkk/zigbee2mqtt#14074] [Devices]
I am trying to pair the SONOFF ZBMINI-L. It is listed as supported in the doco (https://www.zigbee2mqtt.io/devices/ZBMINI-L.html) however when I pair it it comes up as 'Unsupported' and does not appear in HA. I am using Zigbee2MQTT version 1.18.1-1. Has anyone else had this problem?

**Comment by jerrm** [comment]
1.18 is probably too old. Update.

**Comment by Koenkk** [comment]
If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0
